src/crimson/os/seastore/transaction_manager.h

```diff
@@ -176,6 +177,12 @@ class TransactionManager : public SegmentCleaner::ExtentCallbackInterface {
     Transaction &t,
     laddr_t offset);
 
+  /// remove refcount for list of offset
+  using refs_ret = ref_ertr::future<std::list<unsigned>>;
+  refs_ret dec_ref(
+    Transaction &t,
+    std::list<laddr_t> offsets);
```

Comment:
std::vector is almost universally preferred to std::list, especially for a small sequence, and if it is something large and interesting, we should be looking at something like boost::intrusive::list (aka, c++ TAILQ)